ur_driver/src/ur_driver/driver.py

```diff
@@ -467,9 +467,23 @@ def handle(self):
             print "Connection closed (command):", ex
             setConnectedRobot(None)
 
+    def __send_message(self, data):
+        """
+        Send a message to the robot.
+
+        The message is given as a list of integers that will be packed
+        as 4 bytes in network byte order (big endian).
```

Comment:
This might be just me, but *that* seems to be referring to *a list*, which then would mean that the entire list is packed into 4 integers?